src/F2_autoreport.py

```diff
@@ -627,16 +650,16 @@ def create_app(results_json):
 
     simDate = time.strftime("%Y-%m-%d")
 
-    ELAND_LOGO = base64.b64encode(
-        open(
-            os.path.join(REPORT_PATH, "assets", "logo-eland-original.jpg"), "rb"
-        ).read()
+    ELAND_LOGO = encode_image_file(
+        os.path.join(REPORT_PATH, "assets", "logo-eland-original.jpg")
     )
 
-    MAP_STATIC = base64.b64encode(
-        open(os.path.join(REPORT_PATH, "assets", "proj_map_static.png"), "rb").read()
+    MAP_STATIC = encode_image_file(
+        os.path.join(REPORT_PATH, "assets", "proj_map_static.png")
```

Comment:
Can we place adding the map into a `try-exept`? I wanted to generate a report offline yesterday, but of course this could not work. `except` could result eg. in an `[A map of the project location could not be generated.]`

Reply:
The `try/except` is located in the function `encode_image_file`, it will not fail if the path to the map is missing. The map is generated in D0